src/material/dialog/dialog-animations.ts

```diff
@@ -14,6 +14,8 @@ import {
   AnimationTriggerMetadata,
 } from '@angular/animations';
 
+export const _transitionTime = 150;
```

Comment:
@crisbeto I didn't want to copy this value all over the place. Instead I defined this const here, but I'm not confident that this is the right way to do it in this repo. Can you provide some guidance here please?